src/shared/components/contacts/Contacts.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { contacts } from '../../data';
+
+const Contacts = () => (
+    <>
```

Comment:
It's preferred to use `<>`? I think it's quite weird and it's inconsistent when you need to use a `key` prop.

I think we should also add a file of conventions (or only maybe internally) about stuff like this.

Reply:
I prefer `<>` because we avoid confusing Fragment with a react component, which it isn’t. Keys in this context are very rare, and for those cases we may still use Fragment.

Reply:
I also thought it was strange at first, but the ease of use got me and after some time it didn't bother me as much. I also agree with @satazor, I think keys will be an edge-case and maybe we can re-think the logic to embrace this better or, in the last case, use Fragment.

Reply:
-- We should check if there is an eslint rule for this.

Reply:
We have this rule we can use: `jsx-fragments`. 
Where:
![image](https://user-images.githubusercontent.com/11398440/65345396-14cb6580-dbd2-11e9-842b-946c94d29192.png)

Reply:
Done